backend/src/main/java/com/votogether/domain/member/entity/Member.java

```diff
@@ -82,4 +85,18 @@ public void plusPoint(final int point) {
         this.point = this.point + point;
     }
 
+    public void changeNickname(final String nickname) {
+        validateNickname(nickname);
+        this.nickname = nickname;
+    }
+
+    private void validateNickname(final String nickname) {
+        if (nickname.length() < 2 || nickname.length() > 16) {
+            throw new BadRequestException(MemberExceptionType.INVALID_NICKNAME_LENGTH);
+        }
+        if (!Pattern.matches("^[가-힣a-zA-Z0-9]+$", nickname)) {
+            throw new BadRequestException(MemberExceptionType.INVALID_NICKNAME_LETTER);
+        }
+    }
```

Comment:
`P3`
```suggestion
    public void changeNickname(final String nickname) {
        this.nickname = new Nickname(nickname);
    }
```
닉네임 값 객체를 생성하면 해당 로직을 다 값 객체로 위임할 수 있을 것 같은데 어떠신가요?!

Reply:
값 객체 적용해보겠습니다~